libraries/rush-lib/src/logic/operations/CacheableOperationPlugin.ts

```diff
@@ -99,6 +100,25 @@ export class CacheableOperationPlugin implements IPhasedCommandPlugin {
           ? new DisjointSet()
           : undefined;
 
+        for (const operation of recordByOperation.keys()) {
+          if (
+            operation.settings?.allowCobuildWithoutCache &&
+            !cobuildConfiguration?.cobuildWithoutCacheAllowed
+          ) {
+            throw new Error(
+              `Operation ${operation.name} is not allowed to run without the cobuild orchestration experiment enabled. You must use the ${EnvironmentVariableNames.RUSH_COBUILD_WITHOUT_CACHE_ALLOWED} environment variable to enable that mode.`
```

Comment:
There's also an `common/config/rush/experiments.json` file. That may be more ergonomic than an env var.

Reply:
updated in [3b5d30b](https://github.com/microsoft/rushstack/pull/4871/commits/3b5d30be8c4fd801d6a7c4dabd6c553a6532533c)